packages/components/src/components/worker/ecmaversion/client.tsx

```diff
@@ -3,7 +3,7 @@ import { BadgeColorMap, BadgeType } from '../../Badge';
 import { ECMAVersion } from '@rsdoctor/utils/ruleUtils';
 import { master } from 'src/utils/worker';
 
-const getECMAVersionDetectWorker = () => new Worker(new URL('./worker.ts', import.meta.url));
+const getECMAVersionDetectWorker = () => new Worker(new URL('./worker.js', import.meta.url));
```

Comment:
Why do we need to use .js for worker, does module-tools support worker.ts?

Reply:
yes, this './worker.ts' can not compile to .js, so show use .js.

Reply:
@10Derozan cc, can we support this scenario

Reply:
Yes, we recommand use .js when it is a worker.

Reply:
@10Derozan I do not find the document for this